de.wolfvollprecht.UberWriter.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "de.wolfvollprecht.UberWriter",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "uberwriter",
+    "rename-icon": "uberwriter",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--filesystem=host",
+        "--env=IN_FLATPAK=1",
```

Comment:
The project should check if `/.flatpak-info` exists.

Reply:
Ok, didn't know that existed.

Reply:
Done: https://github.com/UberWriter/uberwriter/commit/0cbb9aecf0949537bd6c680cc41a9c049cd2778a , ee7aaa6d8680716f11da46b445a38aa495a015db